oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/ReactiveDelegatingJwtGrantedAuthoritiesConverter.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.oauth2.server.resource.authentication;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.oauth2.jwt.Jwt;
+import reactor.core.publisher.Flux;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+
+/**
+ * Implementation of {@link Converter} that wraps multiple {@link Converter} instances into one.
+ * Reactive version of {@link DelegatingJwtGrantedAuthoritiesConverter}.
+ *
+ * @author rolaca11
+ */
+public class ReactiveDelegatingJwtGrantedAuthoritiesConverter implements Converter<Jwt, Flux<GrantedAuthority>> {
```

Comment:
I appreciate this contribution; however, let's please wait on this until we've got concrete reactive implementations to compose.

The thing is, if an application needs to do a blocking call, then it will need a custom `Converter` already since Reactive doesn't have a concrete one (like `JwtAuthenticationConverter` and `JwtGrantedAuthoritiesConverter` on the servlet side).

Reply:
Does it mean that you want me to delete this class altogether?

Reply:
Yes, please. This may be something we'll need later on if a need is raised for concrete reactive converters.